added few graphic for data visualization

Matas-V opened the pull request:
Added 4 graphs. If you think we need more graphs, please leave a comment with suggestions.
Also removed the commented code.
For me, data removal was not working correctly as lines with empty **Employees** fields were not removed, so I added additional logic to do it next to the graph visualization.

![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/a79ac31b-c844-40a9-97e5-b3db6b7df3a3)

![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/4b799143-11aa-45c3-a44e-d133031088e6)

![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/76c71eb6-2813-4a42-9c46-7fcbcbcd96b7)

![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/c2ea3203-6ae2-4cfb-a7f0-ade1b799cd5e)

Matas-V committed:
added few graphic for data visualization

Matas-V committed:
changed data set for graphs

Ech0ke commented:
- **1 more graph requested** It would be nice to see bar chart of expenses by every state of USA. This would give us more information about company's running costs in different states.

- In **Boxplot of average growth by industry** it would be nice each category to be in a different color. That way it is more appealing to the eye.

Ech0ke reviewed (changes requested):
Changes to be made listed in pr comments

Matas-V committed:
added colors and additional graph

Matas-V commented:
Added colors to boxplot and bar charts.
Added additional graph.

![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/428b7d9f-0131-4dc7-b7ae-9df742455e6a)

![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/071a75ff-948c-4074-b966-071ad7237939)


![image](https://github.com/Ech0ke/data-science-machine-learning/assets/80633152/530b47f8-a214-4a72-8357-6ea3a438dde1)
